src/index.js

```diff
@@ -30,11 +52,17 @@ const init = (auditModifiedClasses, auditAccessClasses = [], options = {}) => {
     const fieldPrefix = options.fieldPrefix || CONFIG_DEFAULTS.fieldPrefix;
     const fieldPostfix = options.fieldPostfix || CONFIG_DEFAULTS.fieldPostfix;
     const parseSDK = options.parseSDK || CONFIG_DEFAULTS.parseSDK;
+    const useMasterKey = options.useMasterKey || CONFIG_DEFAULTS.useMasterKey;
+    const clp = options.clp || CONFIG_DEFAULTS.clp;
     const config = {
-        classPrefix, classPostfix, fieldPrefix, fieldPostfix, parseSDK,
+        classPrefix, classPostfix, fieldPrefix, fieldPostfix, parseSDK, useMasterKey, clp
     };
 
     auditModifiedClasses.forEach((c) => {
+        if (Object.keys(config.clp).length !== 0){
+            parseSDK.Cloud.define(`audit${config.classPrefix}${c}${config.classPostfix}SetCLP`, async req =>  setCLP(c, config));
+            parseSDK.Cloud.run(`audit${config.classPrefix}${c}${config.classPostfix}SetCLP`);
```

Comment:
Hi @cbaker6  - Just curious, why the `parseSDK.Cloud.define`/`parseSDK.Cloud.run` combo? Can't we just call `setCLP(c, config)` directly here?

Reply:
Right, I had this left over from when I was initially testing and couldn't get setCLP to fire.

I've tested without define and run and it works, will make the change in a sec

Reply:
Great, thank you